src/collectors/plugins.d/functions-table.md

```diff
@@ -104,6 +126,10 @@ To define table data, the UI expects this:
 }
 ```
 
+### ${\color{yellow}IMPORTANT}$
+
+On Data values, `timestamp` column value must be in unix micro.
```

Comment:
```suggestion
> **Important Note**
>
> On Data values, `timestamp` column value must be in unix micro.
```

Reply:
changed, although I think it will remove the emphasis it should have

Reply:
If you want to have it as it was go for it, if this is not in Learn I won't gatekeep it, it's just that yellow was not looking good, and the preview style was odd to me